model/fv_regional_bc.F90

```diff
@@ -6892,6 +6898,7 @@ subroutine get_data_source(data_source_fv3gfs,regional)
       if (.not. lstatus) then
        if (mpp_pe() == 0) write(0,*) 'INPUT source not found ',lstatus,' set source=No Source Attribute'
        source='No Source Attribute'
+       call mpp_error(FATAL,'Please add source attribute to IC')
```

Comment:
Do we have a place (like a troubleshooting guide or FAQ) where users who see this message can find out how to solve it?

Reply:
thanks @lharris4 ! that is a good idea. I wonder if @junwang-noaa know whether there is a place we can provide a brief troubleshooting guide for users on how to solve this when there is no source attribute in IC or LBC. We can recommend users to use NCO command "ncatted" to add attribute ad hoc or modify their IC/LBC generation code as a long term solution.

Reply:
Thanks. We may want to start an FAQ and troubleshooting guide for common issues. I believe that GitHub discussions are a good place for this. https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/discussions

Reply:
@Jili-Dong Can you make the sentence more descriptive? e.g. list choices for the source attribute?

Reply:
@junwang-noaa sure. It could changed as "Please add source attribute to IC. The accepted FV3 sources are FV3GFS GAUSSIAN NEMSIO FILE, FV3GFS GAUSSIAN NETCDF FILE or FV3GFS GRIB2 FILE." Please let me know if that looks ok.

Reply:
@junwang-noaa PR updated to add more detailed guide to error message. Thanks for the suggestion!

Reply:
@Jili-Dong - while this error message is better, it doesn't specify which ICs.  I would suggest using a construct for the error message like `fv_regional_bc::get_data_source - input source not found in file <filename> ...`

Reply:
@bensonr error message changed as suggested.

Reply:
I created a section within our discussions labeled ["FAQ and Troubleshooting" ](https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/discussions/categories/faq-and-troubleshooting) Please feel free to contribute any helpful details there.  If we do end up populating this discussion board with helpful information, I will also include reference to this page in the repository's README.

Reply:
Do you have a PR infv3atm and ufs-weather-model associated with this?

Reply:
yes @laurenchilutti 
https://github.com/NOAA-EMC/fv3atm/pull/646
https://github.com/ufs-community/ufs-weather-model/pull/1715

Reply:
> I created a section within our discussions labeled ["FAQ and Troubleshooting" ](https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/discussions/categories/faq-and-troubleshooting) Please feel free to contribute any helpful details there. If we do end up populating this discussion board with helpful information, I will also include reference to this page in the repository's README.

@laurenchilutti I put a more detailed troubleshooting in the discussion board under "FAQ and Troubleshooting":
https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/discussions/264

but it is strange that I can't see the post on the main page:
https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/discussions/categories/faq-and-troubleshooting